test/bigml/test/api/others.clj

```diff
@@ -0,0 +1,92 @@
+;; Copyright 2016 BigML
+;; Licensed under the Apache License, Version 2.0
+;; http://www.apache.org/licenses/LICENSE-2.0
+
+(ns bigml.test.api.others
+  "Contains unit tests for various functions."
+  (:require (bigml.api [utils :as utils]))
+  (:require (bigml.api [core :as api]))
+  (:use clojure.test))
+
+(defn normalized-resource-test 
```

Comment:
For the tests where exceptions are expected, you could use `(is (thrown? ...))` or `(is (thrown-with-msg? ...))`.